src/braket/experimental/autoqasm/__init__.py

```diff
@@ -40,8 +40,8 @@ def my_program():
     result[1] = measure __qubits__[1];
 """
 
-from .api import function  # noqa: F401
-from .instructions import QubitIdentifierType  # noqa: F401
+from .api import function, gate  # noqa: F401
+from .instructions import QubitIdentifierType as Qubit  # noqa: F401
```

Comment:
This was the simplest way to get a user-visible `Qubit` type for use as a type hint in gate definitions. I'm open to suggestions here (or we can defer this until we decide what to do with PhysicalQubit).

Reply:
Looks fine to me 👍